src/database.lua

```diff
@@ -0,0 +1,27 @@
+local sql = require 'lsqlite3'
+
+db = nil -- global database handle
+
+local database = {}
+
+database.init = function (ms)
+    local table_inits = { ms.irc_factoids.init
+                        -- , ms.irc_quotegrabs.init
+                        }
+
+    db = sql.open(ms.config.dbpath)
+    if not db then print('Failed to open the database') end
+
+    for _, v in pairs(table_inits) do v() end
+end
+
+database.cleanup = function (ms)
+    local cleanup_fns = { ms.irc_factoids.cleanup
+                        -- , ms.irc_quotegrabs.cleanup
```

Comment:
please remove this commented line (as quotegrabs haven't been implemented yet) and squash.